examples/realworld/ext/MainPage.js

```diff
@@ -19,11 +19,14 @@ import getFollowedArticles from '@wasp/queries/getFollowedArticles'
 import getAllArticles from '@wasp/queries/getAllArticles'
 import Navbar from './Navbar'
 import ArticleListPaginated from './article/components/ArticleListPaginated'
+import WaspSourceHeader from './WaspSourceHeader'
 
 const MainPage = () => {
   const { data: me } = useAuth()
 
   return (
+    <>
+    <WaspSourceHeader name="Conduit" />
```

Comment:
Let's indent these lines, since they are now all grouped in `<>`? Same for other components in this PR.